notebooks/sam_tests/cmip6_density_transform_pipeline_dev.ipynb

```diff
@@ -8,7 +8,7 @@
     {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #16.    def thin_wrapper(ds):
</pre><p>Is this still needed? I think you can just comment it out and pass <code>preprocess=combined_preprocesssing</code>  below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jbusecke/cmip6_omz/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>